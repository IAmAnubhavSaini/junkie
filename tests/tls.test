#!/bin/sh

NAME="TLS"
PCAPS="pcap/https/mycorp.pcap pcap/https/certificate.pcap pcap/https/tlsv1.pcap pcap/https/mycorp_with_sessions.pcap"
filter() {
	grep -e '^\(TLS\|HTTP\)'
}
OUTPUT="tls.out"
CMDLINE='(tls-add-key "pcap/https/mycorp.pem" "192.168.10.211" "255.255.255.255" #t"http")(tls-add-key "pcap/https/mycorp_with_sessions.pem" "192.168.10.9" "255.255.255.255" #t "http")'

. "$srcdir/dump_test"
