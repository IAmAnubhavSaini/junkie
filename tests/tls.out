-- pcap/https/mycorp.pcap --
TLS: head_len=176, payload=0, version=3.1, content-type=handshake
TLS: head_len=891, payload=0, version=3.1, content-type=handshake, cipher_suite=RSA_WITH_AES_256_CBC_SHA, compression_algo=none
TLS: head_len=198, payload=0, version=3.1, content-type=handshake
TLS: head_len=266, payload=0, version=3.1, content-type=handshake
TLS: head_len=474, payload=0, version=3.1, content-type=data
TLS: head_len=245, payload=0, version=3.1, content-type=data
TLS: head_len=37, payload=0, version=3.1, content-type=alert
TLS: head_len=37, payload=0, version=3.1, content-type=alert
